src/Appwrite/Specification/Format/Swagger2.php

```diff
@@ -270,8 +270,14 @@ public function parse(): array
             );
 
             foreach ($parameters as $name => $param) { // Set params
+                $injections = [];
+
+                if (isset($param['injections'])) {
+                    $injections = array_map(fn ($injection) => $this->http->getContainer()->get($injection), $param['injections']);
+                }
```

Comment:
```suggestion
          $injections = array_map(fn ($injection) => $this->http->getContainer()->get($injection), $param['injections'] ?? []);
```

Maybe? 
I would also generate the spec file after this change and compare the current latest to make sure there is no strange diff 💡